nextflow.config

```diff
@@ -12,6 +12,7 @@ params {
   name = false
   multiqc_config = "$baseDir/assets/multiqc_config.yaml"
   email = false
+  email_on_fail = false
```

Comment:
Should this really be the default?

Reply:
I thought, that's a special option for ppl who wanna set their emails when running the pipeline